stdlib/public/SDK/AppKit/AppKit.swift

```diff
@@ -141,3 +141,10 @@ extension NSWindowController {
     self.dismissController(sender)
   }
 }
+
+extension NSApplication {
```

Comment:
Oh, ok. Then I wonder why some of the other methods here exist, when they could have just been done with APINotes?